www/proxy-sso/src/opnsense/mvc/app/views/OPNsense/ProxySSO/index.volt

```diff
@@ -0,0 +1,174 @@
+                        id: index + '_indicator',
+                        class: 'fa fa-check-circle text-success',
+                    }).appendTo("#" + index);
+                    if(value.message) {
+                        $(".help-block[for='" + index + "']").html(value.message);
+                    }
+                }
+                else if(value.status == "failure") {
+                    jQuery('<div/>', {
+                        id: index + '_indicator',
+                        class: 'fa fa-times-circle text-danger',
+                    }).appendTo("#" + index);
+                    if(value.message) {
+                        $(".help-block[for='" + index + "']").html(value.message);
+                    }
+                }
+                else {
+                    $("#" + index).html(value);
+                }
+
+                if(value.dump) {
+                    jQuery('<div/>', {
+                        id: index + '_dump',
+                        text: htmlDecode(value.dump),
+                        class: 'hidden',
+                    }).appendTo(".help-block[for='" + index + "']");
+                    jQuery('<a/>', {
+                        text: "{{ lang._('Show dump') }}",
+                        href: 'javascript:showDump("' + index + '");',
+                        style: 'padding-left: 20px;',
+                    }).appendTo("#" + index);
+                }
+            });
+        });
+    }
+
+</script>
+
+<div class="alert alert-info hidden" role="alert" id="responseMsg">
+</div>
+
+<ul class="nav nav-tabs" role="tablist"  id="maintabs">
+    <li class="active"><a data-toggle="tab" href="#general"><b>{{ lang._('General') }}</b></a></li>
+    <li><a data-toggle="tab" href="#testing"><b>{{ lang._('Kerberos Authentication') }}</b></a></li>
+</ul>
+
+<div class="tab-content content-box">
+
+    <div class="tab-pane fade in active" id="general">
+        {{ partial("layout_partials/base_form",['fields':generalForm,'id':'frm_GeneralSettings'])}}
+
+        <hr/>
+        <button class="btn btn-primary"  id="applyAct" type="button"><b>{{ lang._('Apply') }}</b> <i id="applyAct_progress" class=""></i></button>
+    </div>
+
+    <div class="tab-pane fade" id="testing">
+
+        {{ partial("layout_partials/base_form",['fields':checkListForm,'id':'frm_CheckList'])}}
+        <hr/>
+        <button class="btn btn-primary" id="RefreshCheckList" type="button"><b>{{ lang._('Refresh') }}</b> <i id="refresh_progress" class=""></i></button>
+
+        {{ partial("layout_partials/base_form",['fields':testingCreateForm,'id':'frm_TestingCreate'])}}
+        <button class="btn btn-primary" id="CreateKeytab" type="button"><b>{{ lang._('Create keytab') }}</b></button>
+        <button class="btn btn-primary" id="DeleteKeytab" type="button"><b>{{ lang._('Delete keytab') }}</b></button>
+        <button class="btn btn-primary" id="ShowKeytab" type="button"><b>{{ lang._('Show keytab') }}</b></button>
```

Comment:
This is the frontend, if possible try clean english here (Key Table for example). This is important for translation and for end users.